packages/@aws-cdk/aws-cloudwatch/README.md

```diff
@@ -160,6 +160,8 @@ The most important properties to set while creating an Alarms are:
 - `evaluationPeriods`: how many consecutive periods the metric has to be
   breaching the the threshold for the alarm to trigger.
 
+To create a cross-account alarm, make sure you have enabled [cross-account functionality](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/Cross-Account-Cross-Region.html) in CloudWatch. Then, set the `account` property in the `Metric` object either manually or via the `metric.attachTo()` function.
```

Comment:
Can you add some description of the `attachTo` function into the readme? Maybe in the Metrics section, not here. It is not explained anywhere in the README right now, only in the doc comment.